RaspberryPi_JetsonNano/python/lib/waveshare_epd/epdconfig.py

```diff
@@ -107,22 +107,34 @@ def module_init(self):
         self.SPI.mode = 0b00
         return 0
 
-    def module_exit(self):
+    def module_exit(self, cleanup=False):
+        '''close SPI handles and optionally cleanup the GPIO pins
+
+        NOTE: Using `cleanup=True` will render the pins unusable until
+        the program restarts.
+
+        Args:
+            cleanup(bool): When True, close all GPIO pins and shutdown interface
+        
+        '''
         logger.debug("spi end")
         self.SPI.close()
 
-        
-        self.GPIO_RST_PIN.off()
-        self.GPIO_DC_PIN.off()
-        self.GPIO_PWR_PIN.off()
+        logger.debug("close 5V, Module enters 0 power consumption ...")
+        if cleanup:
```

Comment:
I'd prefer putting the if statement right before the calls to the close method. It does make sense to turn the pins off because the [waveshare docs](https://www.waveshare.com/wiki/7.3inch_e-Paper_HAT_(F)_Manual) say that keeping the display powered on for prolonged periods of time decreases its lifespan:

"Note that the screen cannot be powered on for a long time. When the screen is not refreshed, please set the screen to sleep mode or power off it. Otherwise, the screen will remain in a high voltage state for a long time, which will damage the e-Paper and cannot be repaired!"

 It's only if you call the close method that they become unusable afterwards.

Reply:
Good catch. I've moved the if statements to only include the close statements.